docs/reference/policy-filtering.md

```
@@ -0,0 +1,224 @@
+          - name: allow-read-tools
+            match: ["kubectl_get_*", "prometheus_*"]
+            when: "True"
+```
+
+Or mount a ConfigMap with your policy configuration.
+
+## Debugging
+
+Policy decisions are logged at INFO level:
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)